Third New England Urbit meetup

7Sages7 opened the pull request:
Added the .md file for the third New England meetup event page per the conditions of my bounty for UF.

7Sages7 committed:
Added the .md file for the third New England mee

page for Events section

ripperi commented on content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md:
```diff
@@ -0,0 +1,8 @@
++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing" starts = "2024-07-27T14:00:00" ends = "2024-07-27T17:00:00" timezone = "America/New_York" location = "Aeronaut Brewing - 14 Tyler St, Somerville, Massachusetts 02143" image = "https://media.urbit.org/site/ecosystem/communities/new-england.jpg" registration_url = "https://www.meetup.com/urbit-new-england-group/events/302151131/" descr ...
+
++++
```
These attributes need to be newline delimited. When using existing events as reference, view them in raw mode first. The markdown preview shows the rendered output, not the raw source content

7Sages7 replied:
fixed. I'll remember this for next time. Thanks.

7Sages7 committed:
Update 2024-07-27-Urbit-New-England-Xthirdmeetup.md

Delimiting lines as @ripperi suggested

7Sages7 reviewed (commented):
fixed. I'll remember this for next time. Thanks.

7Sages7 commented:
Ok. I resubmitted this pull request and incorporated your suggested edit with the delimited lines. Please let me know if you need anything else.

Sent with [Proton Mail](https://proton.me/) secure email.

On Tuesday, July 23rd, 2024 at 6:53 AM, Santeri Hannula ***@***.***> wrote:

> @ripperi commented on this pull request.
>
> ---------------------------------------------------------------
>
> In [content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md](https://github.com/urbit/urbit.org/pull/2326#discussion_r1687851624):
>
>> @@ -0,0 +1,8 @@
> ++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing" starts = "2024-07-27T14:00:00" ends = "2024-07-27T17:00:00" timezone = "America/New_York" location = "Aeronaut Brewing - 14 Tyler St, Somerville, Massachusetts 02143" image = "https://media.urbit.org/site/ecosystem/communities/new-england.jpg" registration_url = "https://www.meetup.com/urbit-new-england-group/events/302151131/" description = "The third meetup for Urbiters in New England" dark = true darken_image = "true"
> +
> ++++
>
> These attributes need to be newline delimited. When using existing events as reference, view them in raw mode first. The markdown preview shows the rendered output, not the raw source content
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/urbit/urbit.org/pull/2326#pullrequestreview-2193639618), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BA6G4YPARIGE3DBTC5MHKWDZNYY3RAVCNFSM6AAAAABLB7RH7GVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCOJTGYZTSNRRHA).
> You are receiving this because you authored the thread.Message ID: ***@***.***>

ripperi commented on content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md:
```diff
@@ -0,0 +1,17 @@
++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing" 
```
Here as well
```
+++
title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing"
```

ripperi replied:
Then this should be good to go!

7Sages7 committed:
Update 2024-07-27-Urbit-New-England-Xthirdmeetup.md

delimited one more line at the top, per instructions. Thank you for your patience.

7Sages7 commented:
Great. I made the fix. Thank you.

Though this is a bit urgent, it's also okay if you are not able to get this on urbit.org until after the event. I just need it on there for the historical record as I am doing this as part of a grant for UF. (All of my RSVPing is done on Landscape and a third-party service, MeetUp, so everything is fine in that regard.).

Sent with [Proton Mail](https://proton.me/) secure email.

On Friday, July 26th, 2024 at 6:41 AM, Santeri Hannula ***@***.***> wrote:

> @ripperi commented on this pull request.
>
> ---------------------------------------------------------------
>
> In [content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md](https://github.com/urbit/urbit.org/pull/2326#discussion_r1692877608):
>
>> @@ -0,0 +1,17 @@
> ++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing"
>
> Then this should be good to go!
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/urbit/urbit.org/pull/2326#discussion_r1692877608), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/BA6G4YJKV4GMAOOVJITAQ3LZOIRW7AVCNFSM6AAAAABLB7RH7GVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEMBRGUZDQMBQGA).
> You are receiving this because you authored the thread.Message ID: ***@***.***>
